lib/hydra/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hydra
-  VERSION = "10.0.1".freeze
+  VERSION = "11.0.0".freeze
```

Comment:
Looks good.  Do you want to make this 11.0.0.rc1 or 11.0.0.beta first? Or are you confident it's GTG?